app/views/calls/_commitment_body.html.erb

```diff
@@ -1,4 +1,3 @@
 <div class="text">
-  <%= commitment.user.first_name %>
-  <%= commitment.body %>
+  <strong><%= commitment.user.first_name %>:</strong>&nbsp;<%= commitment.body %>
```

Comment:
I think we want to use `b` here. `strong` is used to emphasize something semantically, like some important part of a sentence. `b` is more for visual enhancement/emphasis.

Reply:
Understood. Thanks!